web-server/django_web_server/django_web_server/static/stylesheets/website/website.js

```diff
@@ -1,11 +1,97 @@
 
-var map = L.map('mapid').setView([30, 0], 0.5);
+var map = L.map('mapid').setView([60, 10], 3);
```

Comment:
An improvement would be to replace the hardcoded values with constants with explanatory names.